whipper/command/cd.py

```diff
@@ -461,7 +464,10 @@ def _ripIfNotRipped(number):
             start, stop = htoa
             logger.info('found Hidden Track One Audio from frame %d to %d',
                         start, stop)
-            _ripIfNotRipped(0)
+            if not self.options.skip_htoa:
```

Comment:
This check should probably be together before the `if htoa:` line. That way, we could avoid running/calling `self.program.getHTOA()` if we're not going to use it anyway. (And as a consequence, the `logger.info()` should be rephrased.)

Is `htoa` used elsewhere though? For the `.log` file maybe? It should probably say in the .log if the HTOA was deliberately skipped. (Esp. if we decide to still search for HTOA and actually find one.)

Reply:
If the HTOA is ripped to a track, it is logged.  But if it's ripped and discarded as silence, that is not logged.  I think I know how to get info into the log.  I wanted to skip after checking so that the  user knows something is being skipped.  Alternatively, I could just make failure to rip htoa non-fatal, which is why I added the skip-htoa in the first place.  

And I added the signed-off-by tag.

Reply:
But I see that there are commits down the tree there are also missing signed-of-by.

Reply:
Like I said, I would personally just prefer that the `if` check goes before `htoa = self.program.getHTOA()` as there's no reason to call this method if we know we're going to skip it anyway.

> I wanted to skip after checking so that the user knows something is being skipped.

This is a good reason though, but could maybe also just say that "Checking for HTOA was skipped."? @MerlijnWajer or @JoeLametta can maybe provide their thoughts as well.

Reply:
> Is `htoa` used elsewhere though? For the `.log` file maybe? It should probably say in the .log if the HTOA was deliberately skipped. (Esp. if we decide to still search for HTOA and actually find one.)

:+1: 

> This is a good reason though, but could maybe also just say that "Checking for HTOA was skipped."? @MerlijnWajer or @JoeLametta can maybe provide their thoughts as well.

I think it may be useful to report that a silent HTOA was detected and skipped in the logfile too (probably in another pull request).
Regarding to the if location, I'm torn...